config/application.yml.default

```diff
@@ -643,6 +644,7 @@ test:
   sign_in_recaptcha_annotation_enabled: true
   skip_encryption_allowed_list: '[]'
   socure_docv_document_request_endpoint: 'https://sandbox.socure.test/documnt-request'
+  socure_docv_images_request_endpoint: 'https://upload.socure.us/api/5.0/documents/'
```

Comment:
is there a sandbox option? how would i figure that out, it wasn't in the documentation